internal/utils/sha256_hash.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+)
+
+func GetSHA256Hash(content []byte) string {
+	hash := sha256.New()
+	hash.Write(content)
+	return hex.EncodeToString(hash.Sum(nil))
+}
```

Comment:
Is this func necessary given how easy it is to write?

```golang
hex.EncodeToString(sha256.New().Sum(content))
```

Reply:
With the current implementation we also only need to compute the hash in one location so I removed the util function.